app/Http/Controllers/Admin/UserController.php

```diff
@@ -106,7 +111,30 @@ public function store(UserFormRequest $request)
      */
     public function show($id)
     {
-        //
+        $user = $this->userRepository->findOrFail($id);
+
+        $roles = $this->roleRepository->orderBy('id', 'desc')->get();
+
+        if (!empty($roles)) {
+            foreach ($roles as $role) {
+                $flag = DB::table('role_user')
+                        ->where('user_id', $id)
+                        ->where('role_id', $role->id)
+                        ->where('deleted_at', null)
+                        ->first();
+
+                if ($flag != null) {
+                    $role->checked = 1;
+                } else {
+                    $role->checked = 0;
+                }
+            }
+        }
+
+        return view('admin.user.show', [
+            'user'  =>  $user,
+            'roles' =>  $roles
+        ]);
     }
 
     /**
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/UserController.php</br>Expected 1 space between "'user'" and double arrow; 2 found